### PR TITLE
Add links to the new "Automation Examples" and "App Central" articles

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds links to direct readers to these two new articles:
- https://help.sumologic.com/docs/cse/automation-service/automation-service-examples/
- https://help.sumologic.com/docs/cse/automation-service/automation-service-app-central/

Issue number: None

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cse/automation-service/automation-service-integration-framework.md [hunk 1]
```diff
@@ -313,6 +313,8 @@ To export an action, click on the **Export** button below the action name.
 
 ##  Example files
 
+Following are example definition and action files for integrations. To see an explanation of the file contents, see [Integration framework file formats](/docs/cse/automation-service/automation-service-integration-framework#integration-framework-file-formats) above. To see integration definition and action files used in a working integration, see [Configure a custom integration](/docs/cse/automation-service/automation-service-examples#advanced-example-configure-a-custom-integration).
```
Comment (kimsauce) [inline docs/cse/automation-service/automation-service-integration-framework.md:316]:
```suggestion
Following are example definition and action files for integrations. To see an explanation of the file contents, see [Integration framework file formats](#integration-framework-file-formats) above. To see integration definition and action files used in a working integration, see [Configure a custom integration](/docs/cse/automation-service/automation-service-examples#advanced-example-configure-a-custom-integration).
```

File: docs/cse/automation-service/automation-service-playbooks.md [hunk 2]
```diff
@@ -11,13 +11,12 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 
-Playbooks can be configured to execute
-automatically without user intervention, acting on information from the
-incident, or can be executed in interactive mode, where user input is
-required to authorize predefined actions.
+Playbooks can be configured to execute automatically without user intervention, acting on information from the incident, or can be executed in interactive mode, where user input is required to authorize predefined actions.
 
 ## View playbooks
 
+The following procedure describes how to view playbooks already installed in your environment. To add more playbooks, [create a playbook](/docs/cse/automation-service/automation-service-playbooks/#create-a-new-playbook), or [install playbooks from App Central](/docs/cse/automation-service/automation-service-app-central/#install-a-playbook-from-app-central).
```
Comment (kimsauce) [inline docs/cse/automation-service/automation-service-playbooks.md:18]:
```suggestion
The following procedure describes how to view playbooks already installed in your environment. To add more playbooks, [create a playbook](#create-a-new-playbook), or [install playbooks from App Central](/docs/cse/automation-service/automation-service-app-central/#install-a-playbook-from-app-central).
```

File: docs/cse/automation-service/automation-service-playbooks.md [hunk 3]
```diff
@@ -27,6 +26,10 @@ required to authorize predefined actions.
 
 ## Create a new playbook
 
+Before you create your own playbook, first [view playbooks](/docs/cse/automation-service/automation-service-playbooks/#view-playbooks) to make sure there isn't one already that does what you want to accomplish, and also check to see if you can [install a playbook from App Central](/docs/cse/automation-service/automation-service-app-central/#install-a-playbook-from-app-central) that does what you need.
```
Comment (kimsauce) [inline docs/cse/automation-service/automation-service-playbooks.md:29]:
```suggestion
Before you create your own playbook, first [view playbooks](#view-playbooks) to make sure there isn't one already that does what you want to accomplish, and also check to see if you can [install a playbook from App Central](/docs/cse/automation-service/automation-service-app-central/#install-a-playbook-from-app-central) that does what you need.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor fixes

**Comment by jpipkin1** [comment]
Thanks for fixing those oopsies, @kimsauce.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**